app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const { getTopics, sendBadRequest } = require('./controllers/controllers');
+const {
+  handle400,
+  handle404,
+  handle500,
+} = require('./error-handlers/error-handlers');
+
+const app = express();
+app.use(express.json());
```

Comment:
app.use(express.json()) allows our express sever to parse incoming json on a request.body - but since we aren't using this yet it shouldn't be included on a pr at this point